test/test_sparse_csr.py

```diff
@@ -2379,6 +2379,64 @@ def test_sampled_addmm_errors(self, device, dtype):
         with self.assertRaisesRegex(RuntimeError, r"Expected mat2 to have strided layout"):
             torch.sparse.sampled_addmm(a_sparse, a, a_sparse)
 
+    @onlyCPU
+    @dtypes(torch.float32, torch.float64)
```

Comment:
If bfloat16 is supported, can you also add it here?

Reply:
Updated! also add another test case `test_sparse_mm_reduce_sum` to compare `torch.sparse.mm(reduce='sum')` with `torch.mm` for both forward and backward, mathematically them should be equal.